scripts/fast-reboot

```diff
@@ -713,6 +713,11 @@ then
   systemctl stop "$service_name"
 fi
 
+if [[ "$sonic_asic_type" = 'clounix' ]];
+then
+  systemctl stop clx-modules.service || debug "Ignore stopping clounix service error $?"
```

Comment:
should failure be logged as error in syslog instead of debug?

Reply:
Hi kcudnik:
     You're right, "debug" will be recorded in syslog,because in this fast-boot file the definition of "debug" is：
       function debug()
{
    if [[ x"${VERBOSE}" == x"yes" ]]; then
        echo `date` $@
    fi
    logger "$@"
}

Reply:
@kcudnik  
Hi Kamil, 
Thanks for your comment. 
As Qinghua’s mentioned, this log will be logged in syslog, looks like this script is using the similar style.
May I know if we can keep this this change?
Thanks again for your feedback.

Reply:
sure , works for me